vmdb/app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -10,7 +10,7 @@ def textual_group_properties
 
   def textual_group_relationships
     # Order of items should be from parent to child
-    items = %w(ems services containers container_node lives_on)
+    items = %w(ems services containers container_node lives_on replicator)
```

Comment:
Respect the order. It should be:

    items = %w(ems services replicator containers container_node lives_on)